webservice/model/assignment.go

```diff
@@ -186,19 +201,29 @@ func (repo *AssignmentRepository) Insert(assignment Assignment) (int, error) {
 		// Create query string
 		query := "UPDATE assignments SET reviewers = ? WHERE id = ?;"
 		// Prepare and execute query
-		_, err := db.GetDB().Exec(query, assignment.Reviewers, id)
+		_, err := tx.Exec(query, assignment.Reviewers, id)
 		// Check for error
 		if err != nil {
+			log.Fatal(err.Error())
+			tx.Rollback() //quit transaction if error
 			return 0, err
 		}
 	}
 
 	// Set created date
 	query = "UPDATE assignments SET created = ? WHERE id = ?;"
 	// Prepare and execute query
-	_, err = db.GetDB().Exec(query, date, id)
+	_, err = tx.Exec(query, date, id)
 	// Check for error
 	if err != nil {
+		log.Fatal(err.Error())
```

Comment:
No need for logging inside here, cuase error is a return value.
And don't use log.Fatal() cauase it will stop the whole application.